packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -53,40 +53,43 @@ function InteractionLayer ({
     return svgEventOffset
   }
 
+  function createMark (event) {
+    const mark = activeTool.createMark({
+      id: cuid(),
+      frame,
+      toolIndex: activeToolIndex
+    })
+
+    mark.initialPosition(convertEvent(event))
+    setActiveMark(mark)
+    setCreating(true)
+    mark.setSubTaskVisibility(false)
+  }
+
   function onPointerDown (event) {
+    const { target, pointerId } = event
+    target.setPointerCapture(pointerId)
     if (disabled || move) {
       return true
     }
-    let activeMark
 
     if (!activeTool.type) {
       return false;
     }
 
     if (creating) {
-      activeMark = activeTool.handlePointerDown && activeTool.handlePointerDown(convertEvent(event))
-      if (activeMark.finished) setCreating(false)
+      activeTool.handlePointerDown && activeTool.handlePointerDown(convertEvent(event), activeMark)
+      if (activeMark.finished) onFinish(event)
```

Comment:
Yeah, the original code doesn't release pointer capture on finish. That shouldn't be a problem, because the browser should automatically release captured pointers anyway, but I thought it was better to have finished marks handled consistently.